documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -0,0 +1,225 @@
+[[ip-address]]
+=== How to Find your IP Address
+
+To find the local IP of your Raspberry Pi, use one of the following methods.
+
+==== Desktop
+
+Hover over the network icon in the system tray, and a tooltip will appear. This tooltip displays the name of the network you're currently connected to and your IP address.
+
+image::images/network-tooltip.png[the Network Manager tooltip displaying a WiFi network name and IP address]
+
+==== Command Line
+
+Run the following command to output your IP address to the command line:
+
+----
+hostname -I
+----
+
+==== Boot Output
+
+If you use a display with your Raspberry Pi and you boot to the command line instead of the desktop, the boot sequence will include your IP address as one of the last few output messages before your login prompt.
+
+==== Router devices list
+
+In a web browser, navigate to your router's IP address. This is often `http://192.168.1.1`, but you may be able to find it printed on a label on your router. This will take you to a control panel. Then log in using your credentials, which is usually also printed on the router or sent to you in the accompanying paperwork. Browse to the list of connected devices or similar (all routers are different), and you should see some devices you recognise. Some devices are detected as PCs, tablets, phones, printers, etc. so you should recognise some and rule them out to figure out which is your Raspberry Pi. Also note the connection type; if your Raspberry Pi is connected with a wire there should be fewer devices to choose from.
+
+==== Network Manager
+
+You can use the built-in Network Manager CLI (`nmcli`) to access details about your network. Run the following command:
```

Comment:
I wonder if this should be coalesced into the "Command Line" section? Perhaps something like "If you want further information, you can also use Network Manager CLI (`nmcli`) ..." ? :shrug:

Reply:
Debatable. Many of the other sections are *also* command line... just in increasingly complicated ways.

Personally I'm in favor of eliminating all of the sections other than "command line", "desktop", and maybe one alternative for advanced use cases.

Reply:
But I'm also willing to table that discussion out of respect to Scope Creep.

Reply:
Probably a decision for @aallan then :slightly_smiling_face: